{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/utils/context_processors.py

```diff
@@ -2,4 +2,5 @@
 
 
 def settings_context(_request):
-    return {"settings": settings}
+    # Be explicit
```

Comment:
I'm not sure it will make sense outside of the context of this pull request, maybe we could make it a full sentence?

I also suggested a docstring as this function has none currently.

```suggestion
    """Settings available by default to the templates context."""
    # Note: we intentionally do NOT expose the entire settings
    # to prevent accidental leaking of sensitive information
```

Reply:
Ah good point. Committing. Thanks!